pythran/tests/test_numpy_func0.py

```diff
@@ -370,6 +380,36 @@ def test_identity0(self):
     def test_identity1(self):
         self.run_test("def np_identity1(a): from numpy import identity ;return identity(a)", 4, np_identity1=[int])
 
+    def test_fromfile0(self):
+        x = numpy.random.randint(0,2**8,1000).astype(numpy.uint8)
+        x.tofile(self.temp_name)
```

Comment:
you approach is not thread safe. You should create a temporary file per test case, otherwise two test case run in parallel would overwrite each other.